model/model_test.go

```diff
@@ -152,8 +151,7 @@ func (*testModelSuite) TestJobStartTime(c *C) {
 	}
 	t := time.Unix(0, 0)
 	c.Assert(t, Equals, TSConvert2Time(job.StartTS))
-	ret := fmt.Sprintf("%s", job)
-	c.Assert(job.String(), Equals, ret)
+	c.Assert(job.String(), Equals, job.String())
```

Comment:
this test is thus useless, just asserting the pureness of the `String()` implementation. how about just remove it, or change it to the real expected string.

Reply:
Done